config/libc/glibc.in

```diff
@@ -416,4 +417,10 @@ config GLIBC_ENABLE_WERROR
 
 endif
 
+config GLIBC_ENABLE_COMMON_FLAG
+    bool "Enable -fcommon flag for older version of glibc when using GCC >=10"
+    default y if GLIBC_2_29_or_older && GCC_10_or_later
+    default n if GLIBC_2_30_or_later || GCC_9_or_older
```

Comment:
I don't think the `default n` is needed since that is the default.

Reply:
Without the `default n...` option the menuconfig does not automatically turn this flag off once a newer version of glibc is chosen. So a scenario when I choose glibc 2.30 then 2.29 then 2.30 again becomes broken as the -fcommon remains enabled.